src/reg.c

```diff
@@ -422,3 +422,12 @@ const struct sa *reg_laddr(const struct reg *reg)
 
 	return sipreg_laddr(reg->sipreg);
 }
+
+
+const struct sa *reg_paddr(const struct reg *reg)
+{
+	if (!reg)
+		return NULL;
+
+	return sipreg_paddr(reg->sipreg);
```

Comment:
Probably better to save `paddr` locally. Then there is no changes to libre needed ...

Reply:
You're right, that's much better. Done.